tasks/fix-cat2.yml

```diff
@@ -711,12 +711,21 @@
       - RHEL-07-020650
 
 - name: "MEDIUM | RHEL-07-020660 | PATCH | All files and directories contained in local interactive user home directories must be owned by the owner of the home directory."
-  command: "true"
-  changed_when: no
+  block:
+    - name: "MEDIUM | RHEL-07-020660 | AUDIT | All files and directories contained in local interactive user home directories must be owned by the owner of the home directory."
+      command: find {{ item.split(':')[2] }} ! ( -user {{ item.split(':')[0] }} -o -user root )
+      changed_when: no
+      register: rhel_07_020660_audit
+      with_items: "{{ rhel7stig_passwd_homedir_audit.stdout_lines }}"
+
+    - name: "MEDIUM | RHEL-07-020660 | PATCH | All files and directories contained in local interactive user home directories must be owned by the owner of the home directory."
+      #include_tasks: chown_user_file.yml
+      command: chown {{ item.item.split(':')[0] }} {{ item.stdout_lines | join(' ') }}
```

Comment:
Can we use a native Ansible module to do this instead of the command module?

Reply:
We might, but the problem I ran into was that the `find` module won't let select files based off the owner. Thus the `command` version in the audit task, and we end up with using its `stdout_lines`.

Alternative 1: in the audit task, recurse through each home directory with the `find` module and a register, regardless of owning user. In the patch task, iterate over that register and use the `file` module to set ownership on each file found. The issue there is that I think we'd have to decide that all files in the home directory must be owned by the home directory owner, and not root.

Alternative 2: in the audit task, use the command module like I'm doing now to get a list of files not owned by root or the home directory owner, but don't record those in a register. In the patch task, use the shell module to run the same find command with `-print0` instead of the default `-print`, and pass that over a pipe to `xargs -0`. That'll get us the ability to fix files that have spaces in their names, and won't touch any files already owned by root or the home directory owner.

Reply:
Oh I'm fine using `command: find` to find the files to operate on...the Ansible find module lacks a lot and can be rather slow.

I more meant instead of `command: chown` can we use the Ansible file module?

Reply:
Yeah, I think so. Iterate over the find command's stdout_lines for the path attribute of a file task.